decidim-results/app/models/decidim/results/result.rb

```diff
@@ -19,7 +19,7 @@ def commentable?
 
       # Public: Overrides the `accepts_new_comments?` Commentable concern method.
       def accepts_new_comments?
-         commentable? && !feature.active_step_settings.comments_blocked
+        commentable? && !feature.active_step_settings.comments_blocked
```

Comment:
I know this is outside this PR, but: this implementation is the same in all models, maybe we can set it as default at `Commentable`